src/flows/shared/components/WizardFooter/WizardFooter.tsx

```diff
@@ -21,6 +23,7 @@ function WizardFooter({
   shouldHideFooter,
   shouldHideSecondaryButton,
   footerButtonTextMap,
+  isOnboarding,
```

Comment:
`WizardFooter` now takes a prop called `isOnboarding` which will determine whether to show "Back" or "Cancel" in the back button. I don't think we could access this with the prior configuration (unless I missed something)

Reply:
oh dang i should've mentioned that you can use `useWizardId` to check if the `id === 'onboarding'`! so you won't need to use this prop.

example usage: https://github.com/gallery-so/gallery/blob/main/src/flows/shared/steps/OrganizeGallery/OrganizeGallery.tsx#L69

Reply:
Great, thanks!